### PR TITLE
Removes the traitor kill objective

### PR DESCRIPTION
**Pull request by Superhats**
Theoretically removes the _assassinate_ traitor objective, replaces it with marroon
From a gameplay standpoint, both are mostly similar, with marroon giving the traitor more freedom in their strategy, as they aren't forced to kill the person, while still leaving murder as an option.

### CODE REVIEW

**Comment by Superhats** [comment]
Credit for the idea goes to @FeiH#6969

**Comment by KasparoVy** [comment]
Legendary #6969.

**Comment by Superhats** [comment]
:ok_hand:

**Comment by Bram961** [comment]
Could just add maroon instead, I do not see a reason to remove the kill objective.

**Comment by ID404-NotFound** [comment]
Does this stem from people being upset about being assassinated early into round and perma-killed?

**Comment by Superhats** [comment]
In fact, this stems from people being upset about their assassination targets being SSD in unreachable locations/outright missing the whole round.
I do this because assassinate and maroon are fairly similar, with the only difference being that maroon doesn't force you to kill your target. I feel like this would give traitors more choice, in how to approach their targets.

**Comment by Bram961** [comment]
> In fact, this stems from people being upset about their assassination targets being SSD in unreachable locations/outright missing the whole round.

This would not be fixed by replacing it with a maroon objective. the only change this would cause there is green letters instead of red ones at the end of the shift. it's still just as boring for the traitor and we would much prefer if someone ahelped if they were unable to find their target anywhere for an extended period of time so we can check.

**Comment by FlattestGuitar** [comment]
>This would not be fixed by replacing it with a maroon objective. the only change this would cause there is green letters instead of red ones at the end of the shift. it's still just as boring for the traitor and we would much prefer if someone ahelped if they were unable to find their target anywhere for an extended period of time so we can check.

That's a bit silly. How often are players supposed to ahelp when they can't find their target at their workplace? I recall telling played to sod off *myself*, solely because this can't be a reasonable expectation from neither the players nor the staff team.

* the problems brought up by the description could easily be fixed by making the target switch after 15 minutes of your objective being afk or SSD, whichever you guys feel like checking
* I think giving antags the choice of killing people or just trapping them somewhere or striking a deal are a far bigger consequence (and a good thing!) coming from this PR, and it should be discussed more thoroughly.

**Comment by ID404-NotFound** [comment]
Isn't there also already an objective that is just make sure X doesn't escape on the escape shuttle or pods?

File: code/game/gamemodes/traitor/traitor.dm [hunk 1]
```diff
@@ -138,11 +138,6 @@
 					maroon_objective.owner = traitor
 					maroon_objective.find_target()
 					traitor.objectives += maroon_objective
-				else
```
Comment (Fox-McCloud) [inline code/game/gamemodes/traitor/traitor.dm]: This won't even work well code wise.

Comment (Crazylemon64) [reply code/game/gamemodes/traitor/traitor.dm]: Ah, yes - if you don't guarantee an objective is added inside the loop, strange things will happen in terms of objective assignment - resulting in less objectives being assigned overall.

**Comment by Crazylemon64** [comment]
After a discussion, we have come to the conclusion this change is too large to make. We have concluded that this PR shall be closed.

**Pull request closed without merging**